src/cli/execute.js

```diff
@@ -0,0 +1,20 @@
+const meow = require('meow')
+
+const scanFromFile = require('../scanFromFile')
+
+const helpText = require('./helpText')
+const flags = require('./flags')
+
+const options = { flags }
+const cliDefault = meow(helpText, options)
```

Comment:
I think there's no need to mock the meow cli. we can test just overwriting `process.argv` variable.

Reply:
I actually tried few things and couldn't imitate to `process.argv`. My final thought was testing the logic you wrote, not the part taken from `meow`.

similar issue also discussed [here](https://stackoverflow.com/questions/33878230/how-to-unit-test-a-tool-that-uses-command-line-args)

Reply:
Oh, yes, unit tests are to be implemented yet, that's why I prefer to use meow with as less mocks as possible.

Also, you can't mock process.argv because it's obtained when you start meow(). Wrapping it inside a function would make it possible to edit process.argv.